QUEST_LV_0100.tsv

```diff
@@ -530,8 +530,8 @@ QUEST_LV_0100_20150317_000529	Soldiers are reporting that other monsters are app
 QUEST_LV_0100_20150317_000530	Those silly things keep coming to us.{nl}Now may be the only chance we have to go and get Sparnas.{nl}Tell me when you are ready.
 QUEST_LV_0100_20150317_000531	They are just creating so many problems.{nl}They even got their hands on the supplies, I will not feel enough even if I hit him with a bat in the middle of the market.
 QUEST_LV_0100_20150317_000532	What? Tutu?{nl}I don't have time to think about those insignificant critters.
-QUEST_LV_0100_20150317_000533	When you level up, you can increase a status of your choice and become stronger. {nl} Would you like to check it now?
-QUEST_LV_0100_20150317_000534	It's important to think ahead about what you would want to be. {nl} It's never an easy thing.
+QUEST_LV_0100_20150317_000533	$When you level up, you can increase an attribute of your choice and become stronger. {nl} Would you like to try doing so now?
```

Comment:
Attribute = passive skill

Status is correct. Status is: STR, DEX, INT, SPR and CON. I've tried changing this, but IMC wants it to remain Status.